### PR TITLE
[@types/google-apps-script] Add Drive API V3

### PR DESCRIPTION
**Pull request by sinsky**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://developers.google.com/drive/api/reference/rest/v3>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

---

Drive API V2 is still active. Therefore, use the `Drive_V3` namespace instead of `Drive` for V3.

However, when using DriveAPI with Google AppsScript, the default can be called with `Drive` regardless of V2 or V3.
I am wondering if it is okay to change to `Drive_V3` based on my opinion. I would be happy to receive any advice.

### CODE REVIEW

**Comment by mtgto** [comment]
This is a great works. Thank you for your contribution.
Could you please tell me how you obtained this type definition so that I can review this Pull Request?

Besides, you, [as you have commented](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69023#issuecomment-2002529211), should write the test for this pull request.

----

> However, when using DriveAPI with Google AppsScript, the default can be called with Drive regardless of V2 or V3.
> I am wondering if it is okay to change to Drive_V3 based on my opinion. I would be happy to receive any advice.

It seems to use DriveV3 as default 🎉 

Can you rename `Drive` to `Drive_v2` in drive_v2.d.ts?

```ts
// drive_v2.d.ts
declare var Drive_v2: GoogleAppsScript.Drive;

// drive_v3.d.ts
declare var Drive: GoogleAppsScript.Drive_V3;
```

**Comment by sinsky** [comment]
@mtgto Thank you!

> Could you please tell me how you obtained this type definition so that I can review this Pull Request?

Sorry, I looked for it too, but there was no official documentation.
So I checked the network from the AppsScript editor and found something like a type definition, so I created this pull request. (maybe by monaco editor?).
What should I do in this case?

AppsScript - DevTool network

> Besides, you, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69023#issuecomment-2002529211, should write the test for this pull request.

I was wondering if I should add the other APIS libraries since they had not been added.
I will add a test later.

> > However, when using DriveAPI with Google AppsScript, the default can be called with Drive regardless of V2 or V3.
> > I am wondering if it is okay to change to Drive_V3 based on my opinion. I would be happy to receive any advice.
> > 
> It seems to use DriveV3 as default 🎉
> 
> Can you rename Drive to Drive_v2 in drive_v2.d.ts?
> 
> ```ts
> // drive_v2.d.ts
> declare var Drive_v2: GoogleAppsScript.Drive;
>   
> // drive_v3.d.ts
> declare var Drive: GoogleAppsScript.Drive_V3;
> ```

Thanks! It is indeed a good idea.
I will make the changes with that content.

File: types/google-apps-script/apis/drive_v3.d.ts [hunk 1]
```diff
@@ -0,0 +1,932 @@
...
+                    resource: Drive_V3.Schema.ModifyLabelsRequest,
+                    fileId: string,
+                ): Drive_V3.Schema.ModifyLabelsResponse;
+                remove(
+                    fileId: string,
+                ): void;
+                remove(
+                    fileId: string,
+                    optionalArgs: object,
+                ): void;
```
[the remark this answers is not in the document]
Comment (mark-monteiro) [reply types/google-apps-script/apis/drive_v3.d.ts:205]: Apologies, I commented here before actually testing this myself. The function name used in Apps Script is in fact `remove` even though it is documented in the API as `delete`. You may disregard this suggestion

**Comment by sinsky** [comment]
I wanted to make some updates, so I submitted another pull request. I will close this one.

**Pull request closed without merging**